repose-aggregator/functional-tests/spock-functional-test/src/test/groovy/framework/mocks/MockValkyrie.groovy

```diff
@@ -76,10 +76,12 @@ class MockValkyrie {
     String account_perm = "also_butts"
     String contact_id = ""
     String tenant_id = ""
+    Integer fixnumber = 10
+    Integer multiplier = 1
 
```

Comment:
If this is going to be split into two separate values, then the names should probably reflect what they are tied to (e.g. device, inventory). For example deviceCount/invnentoryCount, devMulti/invMulti, device_multiplier/inventory_multiplier.